O/OpenSpiel/build_tarballs.jl

```diff
@@ -57,8 +57,11 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
+    BuildDependency(PackageSpec(name="libjulia_jll", version=julia_version)),
     Dependency("libcxxwrap_julia_jll")
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version = v"8")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
+    preferred_gcc_version=v"10",
```

Comment:
Using GCC 10 is a fairly bad idea in terms of compatibility

Reply:
Agree, I'll fix it soon.

Still testing things 😭